cmd/mungedocs/unversioned_warning_test.go

```diff
@@ -99,7 +99,7 @@ refer to the docs that go with that version.
 <!-- TAG RELEASE_LINK, added by the munger automatically -->
 <strong>
 The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.1/filename.md).
+[here](http://releases.k8s.io/release-1.2/filename.md).
```

Comment:
would it be really hard to make it use `latestReleaseBranch` instead of having to do this part every release?